src/reducers/entities/channels.js

```diff
@@ -492,6 +501,22 @@ function totalCount(state = 0, action) {
     }
 }
 
+export function manuallyUnread(state = {}, action) {
+    switch (action.type) {
+    case ChannelTypes.REMOVE_MANUALLY_UNREAD: {
+        const {[action.data.channelId]: removeThis, ...newState} = state;
```

Comment:
neat, took me a couple of seconds to understand but I do like it

Reply:
I'll add a comment then, I don't want people struggling with 'smart code'

Reply:
I'd really prefer if this was written out in a way that didn't require a comment to explain it. Even if we can understand something like this, we have to remember that we're going to have others reading this code, including community members who may be less experienced with JavaScript's quirks. It might save a few lines to do it this way compared to the more repetitive way that other reducers do it, but I don't think that's worth it for the confusion that it might cause in the future.

Reply:
no problem, I'll change it to a delete statement, I like it less not for brevity, but for mutating stuff.